### PR TITLE
Remove taxon icons

### PR DESCRIPTION
**Pull request by Matt-Yorkley**
#### What? Why?

These icons are inconsistently used across instances and have already started to be removed in a few places ( for example in the mobile UX redesign). They regularly trigger a long string of 404 responses in some servers, but it's not even consistent :see_no_evil: 

:fire: :fire:

#### What should we test?

Taxon icons are not present in the /shops index page, or on the /shop page. Admin (superadmin?) pages related to editing taxons are working correctly (view, update, etc).

#### Release notes

Removed taxon icons

### CODE REVIEW

**Comment by sauloperez** [comment]
Agree but does the product team and the affected instance managers know about it?

**Comment by luisramos0** [comment]
Can we have a screenshot of before and after to see what icons are being removed?
Yeah, this should have product input. We should ask for that but include an alternative with cost: how difficult it is to keep these icons and make them work consistently?

**Comment by RachL** [comment]
@luisramos0 I believe they are only working for AU and UK currently. And maybe one day we will want to provide shops with entries by product category. So we won't use icons but images. 
So I wouldn't waste time on them.

**Comment by sigmundpetersen** [comment]
Isn't this what is already done by @mkllnk through #4671 / #5314 ?

**Comment by mkllnk** [comment]
My work removed the icon from the product listing (https://github.com/openfoodfoundation/openfoodnetwork/pull/5314/commits/e1422e132b379a01a6fdef9e8bd5d123989efaa9). But this pull request is removing the whole feature including the upload of icons etc.

I would like to know what the product team thinks about this, otherwise great initiative!

**Comment by Matt-Yorkley** [comment]
Rebased. The build had some weird EnterpriseController errors... :+1:

**Comment by sauloperez** [comment]
That's what broke the build and @luisramos0 fixed, I believe.

File: app/views/shops/_fat.html.haml [hunk 1]
```diff
@@ -10,7 +10,6 @@
       .trans-sentence
         %div
           %span.fat-taxons{"ng-repeat" => "taxon in hub.taxons"}
-            %render-svg{path: "{{taxon.icon}}"}
```
Comment (luisramos0) [inline app/views/shops/_fat.html.haml:13]: this is removing the icons from the shops page, I think we need a screenshot with before/after so we can see the impact.

Comment (sauloperez) [reply app/views/shops/_fat.html.haml:13]: yes, seeing we don't mess up with margins and padding would be good

Comment (Matt-Yorkley) [reply app/views/shops/_fat.html.haml:13]: This PR vs Katuma prod:

**Filters in /shops**

Screenshot from 2020-07-14 12-05-12
Screenshot from 2020-07-14 12-05-50

**Filters in shop page**

Screenshot from 2020-07-14 12-07-23
Screenshot from 2020-07-14 12-07-01

No change, these icons are already missing on most servers.

**Comment by luisramos0** [comment]
ok, these icons on the top right will also go, correct?
uk
image
AUS
image

**Comment by sauloperez** [comment]
I think so @luisramos0 this is how it looks in Katuma

Screenshot from 2020-07-15 17-02-25

We could as well remove that `taxon-flag` altogether. It won't have any use.

cc @yukoosawa @daniellemoorhead

**Comment by mkllnk** [comment]
The ribbon is already removed in the new product listing branch.

**Comment by filipefurtad0** [comment]
Hi @Matt-Yorkley ,

This looks good,

- /shops page:


Before this PR 

image

After this PR

image


- /shop page:


before this PR

image

after this PR

image


Checked Admin/Superadmin pages related to editing taxons: all works as before.

Ready to go.

**Pull request merged**